sport/app/football/controllers/MatchController.scala

```diff
@@ -12,10 +12,14 @@ import play.api.libs.json._
 import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents}
 import conf.Configuration
 import football.model.{DotcomRenderingFootballMatchSummaryDataModel, GuTeamCodes}
+import play.api.libs.ws.WSClient
+import renderers.DotcomRenderingService
+import services.dotcomrendering.{FootballPagePicker, FootballSummaryPagePicker, RemoteRender}
 
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
 import scala.concurrent.Future
+import scala.concurrent.Future.successful
```

Comment:
Because you are doing `Future.successful` this import isn't seemed to be used anywhere in this file, so we can either remove the import or change `Future.successful` to `successful`